src/views/DelayServer.js

```diff
@@ -0,0 +1,86 @@
+import React, { Fragment, useState } from 'react';
+import axios from 'axios';
+import { useEnvironment } from '../contexts';
+import { Button, TextField, Box, makeStyles, Snackbar, Chip, FormControl } from '@material-ui/core';
+import MuiAlert from '@material-ui/lab/Alert';
+
+const useStyles = makeStyles((theme) => ({
+    inputBox: {
+        marginRight: theme.spacing(1),
+        minWidth: 300
+    }
+}))
+
+export const DelayServer = () => {
+    const [hostname, setHostname] = useState('')
+    const [success, setSuccess] = useState(false)
+    const [error, setError] = useState("")
+    const environment = useEnvironment();
+    const auth = localStorage.getItem('zmt-token')
+    const classes = useStyles()
+
+    const saveServerHostname = async () => {
+        if (hostname === "") {
+            setError("Hostname cannot be empty.")
+            return
+        }
+
+        await axios({
+            url: `${environment.restApiLocation}/admin`,
+            method: 'POST',
+            headers: {
+                'Authorization': auth
+            },
+            params: {
+                'action': 'set_delay_server',
+                'target': hostname
+            }
+        }).then(() => {
+            setSuccess(true)
+        }).catch((e) => {
+            setError(e.message)
+        })
+
+
+    }
+    return (
+        <Fragment>
+            <Box variant="h4" fontWeight="fontWeightBold" fontSize='h6.fontSize' >
+                <Box>
+                    Change the delay server hostname (equivalent of <Chip label="iadmin set_delay_server" />):
+                </Box>
+
+                <Box display='flex' alignItems="center">
+                    <FormControl>
+                        <TextField
+                        className={classes.inputBox}
+                        id="delay_server"
+                        label="example.org"
+                        variant="outlined"
+                        margin="dense"
+                        required
+                        autoFocus
+                        size='medium'
+                        value={hostname}
+                        onChange={(e) => setHostname(e.target.value)}
+                        />  
+                    </FormControl>
+
+                    <Button variant="outlined" color="primary" onClick={saveServerHostname}>
+                        Save
+                    </Button>
+                </Box>
+            </Box>
+
+
+            <Snackbar open={success} autoHideDuration={5000} onClose={() => { setSuccess(false); }}><MuiAlert elevation={6} variant="filled" severity="success">Success! Delay server &quot;{hostname}&quot; was saved.</MuiAlert></Snackbar>
```

Comment:
"Success!" .... do we have exclamation points elsewhere in the app?

Reply:
yep:
<img width="673" alt="Screenshot 2023-06-26 at 6 28 03 PM" src="https://github.com/irods/irods_client_zone_management_tool/assets/62436772/4dd014fa-bcce-4fb4-baeb-e622b1c15d83">